doc/main/tbb_userguide/Controlling_Chunking_os.rst

```diff
@@ -42,7 +45,8 @@ The grainsize sets a minimum threshold for parallelization. The
 ``parallel_for`` in the example invokes ``ApplyFoo::operator()`` on
 chunks, possibly of different sizes. Let *chunksize* be the number of
 iterations in a chunk. Using ``simple_partitioner`` guarantees that
-[G/2] <= *chunksize* <= G.
+[G/2] <= *chunksize* <= G (a chunk will be between half to all of the
```

Comment:
@aleksei-fedotov , `[G/2]` is not always low limit for proportional split, is not it? In some corner case, can the low limit be `[G/3]`?

Reply:
Yes, you are right. With proportional division the guarantees for `simple_partitioner` would be [G/3] <= *chunksize* <= G. However, `simpler_partitioner` is too simple to be aware of proportional split. Therefore, the original statement is correct.